.github/workflows/managed-format-java.yaml

```diff
@@ -0,0 +1,36 @@
+name: Check formatting of java code under managed
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - .github/workflows/managed-format-java.yaml
+      - managed/**/*.java
+
+  pull_request:
+    branches:
+      - master
+    paths:
+      - .github/workflows/managed-format-java.yaml
+      - managed/**/*.java
+
+
+jobs:
+  formatting:
+    name: Check Formatting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 8
+
+      - name: Check formatting
+        run: sbt javafmtCheck
```

Comment:
`sbt javafmtCheck` just checks the files in the src directory. We need to run `sbt javafmtCheckAll` for including test files.
Also, what are your thoughts on running the formatter explicitly during the push to master?
Also, as a follow up, can we make doing an `sbt compile` as part of our build process? We currently have no checks for our managed code build.

Reply:
It isn't easy to add a compile step actually, because of the client jar. To prepare the client jar, we would need to run ./yb_build.sh --java-only (just mvn isn't sufficient because we need protobuf declarations) and that needs more setup. Let's start with this for now.